tests/FakeItEasy.Tests/FakeItEasy.Tests.csproj

```diff
@@ -18,4 +18,8 @@
     <ProjectReference Include="..\..\src\FakeItEasy\FakeItEasy.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="IoC\" />
+  </ItemGroup>
+

```

Comment:
Remove this ;)

Reply:
Yes! I don't know why or how it got added. Sorry I missed it.

Reply:
It was probably added when you removed the only file this folder contained. MSBuild doesn't implicitly include empty folders, so VS added it explicitly so that it remains visible in the project.